layouts/sdk.vue

```diff
@@ -13,7 +13,7 @@ const navigation = computed(() => nav!.value.find((item) => item._path.startsWit
         <UAside>
           <UNavigationTree
             :links="mapContentNavigation(navigation)"
-            default-open
+            default-open-all
```

Comment:
Let's leave the links setting to default-open for now. It is harder to read the fully expanded navigation.

Reply:
Done